Production/govuk_ios/ViewControllers/SettingsViewController.swift

```diff
@@ -21,5 +23,66 @@ class SettingsViewController: BaseViewController,
         super.viewDidLoad()
 
         view.backgroundColor = UIColor.govUK.fills.surfaceBackground
+
+        title = viewModel.title
+
+        navigationController?.setNavigationBarHidden(false, animated: false)
+        navigationController?.navigationBar.prefersLargeTitles = true
+
+        scrollview.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(scrollview)
+
+        addContent()
+
```

Comment:
really minor but theres just a couple of random line breaks in the view did load